tokens-accounting/src/operations.rs

```diff
@@ -68,6 +77,7 @@ pub enum TokenAccountingUndo {
     LockSupply(LockSupplyUndo),
     FreezeToken(FreezeTokenUndo),
     UnfreezeToken(UnfreezeTokenUndo),
+    ChangeTokenAuthority(ChangeTokenAuthorityUndo),
```

Comment:
Does this end up in the storage? If so, do we need to bump the version? My guess is that adding an enum variant at the end is fine but it's good to double check.

Reply:
We already bumped in another PR.